backend/src/main/kotlin/blue/mild/covid/vaxx/error/ExceptionHandling.kt

```diff
@@ -45,6 +40,11 @@ fun Application.installExceptionHandling() {
             logger.warn { cause.message }
             call.errorResponse(HttpStatusCode.BadRequest, "Bad request. ${cause.message}")
         }
+
```

Comment:
Tady bychom meli zacgytavat i DB exception (kvuli prekroceni delky stringu, atd. a hazet 400 misto 500)

Reply:
jo, dobrej napad, netusim, jaka exception se vyhazuje z databaze tbh, urcite to ale chyti ta `Exception`

Reply:
takze bych to zatim nechal byt